home.hbs

```diff
@@ -82,6 +74,10 @@
             <img class=" mt-4 mr-lg-5 img-fluid" src="{{asset "images/wpch_defund.jpg"}}" alt="protest posters on Winnipeg police headquarters" width="432" height="516 ">
             <img class=" mt-4 ml-lg-5 img-fluid mb-200px mb-0px-mobile" src="{{asset "images/wpch_posters.jpg"}}" alt="three police abolition posters on a bridge" width="432" height="432">
         </div>
+
+        <div data-aos="fade-up"  style="display: flex;justify-content: center;margin-bottom: 100px;margin-top: 50px;"> 
```

Comment:
This can be a future refactoring but this pattern of inline CSS for the buttons will now be repeated three times which I think is a good moment to extract it into a class 😯